### ISSUE TITLE
Unable to start plugin with MM 5.10 on Mac

### ISSUE THREAD

**Issue by pclaisse** [moussetc/mattermost-plugin-giphy#16]
Didnt get the plugin (v0.2.0-alpha) to start with mattermost 5.10 on Mac (darwin-amd64).

Error msg: 
```
{"level":"error","ts":1557837166.662481,"caller":"mlog/log.go:172","msg":"Unable to activate plugin","plugin_id":"com.github.moussetc.mattermost.plugin.giphy","error":"unable to start plugin: com.github.moussetc.mattermost.plugin.giphy: fork/exec plugins/com.github.moussetc.mattermost.plugin.giphy/plugin: exec format error","errorVerbose":"fork/exec plugins/com.github.moussetc.mattermost.plugin.giphy/plugin: exec format error\nunable to start plugin: com.github.moussetc.mattermost.plugin.giphy\ngithub.com/mattermost/mattermost-server/plugin.(*Environment).Activate\n\t/go/src/github.com/mattermost/mattermost-server/plugin/environment.go:216\ngithub.com/mattermost/mattermost-server/app.(*App).SyncPluginsActiveState\n\t/go/src/github.com/mattermost/mattermost-server/app/plugin.go:95\ngithub.com/mattermost/mattermost-server/app.(*App).InitPlugins\n\t/go/src/github.com/mattermost/mattermost-server/app/plugin.go:126\ngithub.com/mattermost/mattermost-server/app.(*Server).RunOldAppInitalization.func6\n\t/go/src/github.com/mattermost/mattermost-server/app/server_app_adapters.go:147\ngithub.com/mattermost/mattermost-server/config.(*emitter).invokeConfigListeners.func1\n\t/go/src/github.com/mattermost/mattermost-server/config/emitter.go:35\nsync.(*Map).Range\n\t/usr/local/go/src/sync/map.go:333\ngithub.com/mattermost/mattermost-server/config.(*emitter).invokeConfigListeners\n\t/go/src/github.com/mattermost/mattermost-server/config/emitter.go:33\ngithub.com/mattermost/mattermost-server/config.(*commonStore).set\n\t/go/src/github.com/mattermost/mattermost-server/config/common.go:80\ngithub.com/mattermost/mattermost-server/config.(*FileStore).Set\n\t/go/src/github.com/mattermost/mattermost-server/config/file.go:97\ngithub.com/mattermost/mattermost-server/app.(*Server).UpdateConfig\n\t/go/src/github.com/mattermost/mattermost-server/app/config.go:53\ngithub.com/mattermost/mattermost-server/app.(*App).UpdateConfig\n\t/go/src/github.com/mattermost/mattermost-server/app/config.go:59\ngithub.com/mattermost/mattermost-server/app.(*App).EnablePlugin\n\t/go/src/github.com/mattermost/mattermost-server/app/plugin.go:245\ngithub.com/mattermost/mattermost-server/api4.enablePlugin\n\t/go/src/github.com/mattermost/mattermost-server/api4/plugin.go:187\ngithub.com/mattermost/mattermost-server/web.Handler.ServeHTTP\n\t/go/src/github.com/mattermost/mattermost-server/web/handlers.go:167\ngithub.com/mattermost/mattermost-server/vendor/github.com/gorilla/mux.(*Router).ServeHTTP\n\t/go/src/github.com/mattermost/mattermost-server/vendor/github.com/gorilla/mux/mux.go:162\ngithub.com/mattermost/mattermost-server/vendor/github.com/rs/cors.(*Cors).Handler.func1\n\t/go/src/github.com/mattermost/mattermost-server/vendor/github.com/rs/cors/cors.go:207\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:1995\ngithub.com/mattermost/mattermost-server/app.(*RateLimiter).RateLimitHandler.func1\n\t/go/src/github.com/mattermost/mattermost-server/app/ratelimit.go:107\nnet/http.HandlerFunc.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:1995\nnet/http.serverHandler.ServeHTTP\n\t/usr/local/go/src/net/http/server.go:2774\nnet/http.(*conn).serve\n\t/usr/local/go/src/net/http/server.go:1878\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1337"}
```

Specs:
- Mattermost: 5.10 (Docker Version)
- OS: Mac OS 10.14.4

**Comment by moussetc** [comment]
I tried this plugin version on MM 5.10 Linux and it doesn't work either so compatibility was broken since 5.9.
There will be a new release soon™ which will, among other things, fix this compatibility issue.
Thanks for the report!

**Comment by moussetc** [comment]
The new release is available for MM 5.10 compatibility (and up).
